### PR TITLE
IT-2308: Add suppress rule for CIS1.4

### PR DESCRIPTION
**Pull request by xschildw**
This PR suppresses the findings for CIS1.4 (rotate IAM keys < 90 days) for bootstrap accounts and any service accounts.
Human accounts (synapseprod/dev/dw) are being decomissioned and the associated findings will disappear as they are.

### CODE REVIEW

File: org-formation/075-security-hub/security-hub-suppress-infra.yaml [hunk 1]
```diff
@@ -382,3 +382,39 @@ Resources:
...
+    Type: AWS::Events::Rule
+    Properties:
+      Description: SecHubSuppress findings (identified by generatorId) for the bootstrap and essentials buckets in all accounts
+      EventPattern:
+        detail:
+          findings:
+            Resources:
+              Id:
+                # synapsedw IT-2308, do not exempt human users
+                - prefix: 'arn:aws:iam::383874245509:user/bootstrap-AWSIAMTravisUser-'
```
Comment (xschildw) [inline org-formation/075-security-hub/security-hub-suppress-infra.yaml:397]: Is this account deprecated? Looking it either does not exist or has not been used in a long while (except scicomp/scipool).

Comment (zaro0508) [reply org-formation/075-security-hub/security-hub-suppress-infra.yaml:397]: i believe that is the `org-sagebase-synapsedw` account.  in any case `bootstrap-AWSIAMTravisUser-*` is old and should not be used anymore.  if it's still around then it should probably just be deleted

File: org-formation/075-security-hub/security-hub-suppress-infra.yaml [hunk 2]
```diff
@@ -382,3 +382,39 @@ Resources:
           - SecurityHubFindingsQueue
           - Arn
         Id: Target0
+
+  # This rule suppresses findings with the given generator IDs only for the bootstrap account and service accounts
+  SuppressFindingsForPublicBucketsRule:
+    Type: AWS::Events::Rule
+    Properties:
+      Description: SecHubSuppress findings (identified by generatorId) for the bootstrap and essentials buckets in all accounts
```
Comment (zaro0508) [inline org-formation/075-security-hub/security-hub-suppress-infra.yaml:390]: seems like this is doing more than just boostrap and essentials?

File: org-formation/075-security-hub/security-hub-suppress-infra.yaml [hunk 3]
```diff
@@ -382,3 +382,39 @@ Resources:
           - SecurityHubFindingsQueue
           - Arn
         Id: Target0
+
+  # This rule suppresses findings with the given generator IDs only for the bootstrap account and service accounts
+  SuppressFindingsForPublicBucketsRule:
```
Comment (zaro0508) [inline org-formation/075-security-hub/security-hub-suppress-infra.yaml:387]: this name doesn't seem to match what's being supressed?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/075-security-hub/security-hub-suppress-infra.yaml (hunk 3 above)
Comment (xschildw) [reply org-formation/075-security-hub/security-hub-suppress-infra.yaml:387]: Fixed

File: org-formation/075-security-hub/security-hub-suppress-infra.yaml (hunk 2 above)
Comment (xschildw) [reply org-formation/075-security-hub/security-hub-suppress-infra.yaml:390]: Put a more generic description.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**